tests/utils/ssh/ssh.go

```diff
@@ -33,3 +36,16 @@ var sshIdentity = []string{strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[0], stri
 func InvokeCommand(ip, cmd string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(sshIdentity, "root@"+ip, cmd)...).CombinedOutput()
 }
+
+// InvokeCommandLocally - can be consumed by test directly to invoke
+// any command on the remote host.
+// remoteHostIP: remote machine address to execute on the machine
+// cmd: A command string to be executed on the remote host as per
+// remoteHostIP value
+func InvokeCommandLocally(cmd string) string {
+	out, err := exec.Command("sh", "-c", cmd).CombinedOutput()
+	if err != nil {
+		log.Fatalf("error => %v", err)
```

Comment:
Suggest to include the cmd:
    log.Fatalf("Failed to invoke command [%s]: %v", cmd, err)